_topic_map.yml

```diff
@@ -2845,15 +2845,21 @@ Topics:
   - Name: Splitting traffic between revisions
     File: splitting-traffic-between-revisions-knative
 # Knative Eventing
-- Name: Event workflows
+- Name: Knative Eventing workflows
```

Comment:
Why is it callen 'Knative Eventing workflows' but 'Knative Serving' in the navigation bar ?

![image](https://user-images.githubusercontent.com/99080/112119021-42c1a980-8bbd-11eb-973f-3a4e3f4fe608.png)

Would it make sense to add a top-level "Knative Eventing" section with workflows, source, apache kafka to be part of this ?

I know its not in the scope of this PR, please point me to a place where I can raise my concerns about the different level of granularity that appears on the same level in the TOC. 

Please ignore this comment in the context of this PR.

Reply:
I will open issues in this repo, with [srvls] as prefix in the title (guess this is the correct way to do it ?)

Reply:
--> https://github.com/openshift/openshift-docs/issues/30747

Reply:
@rhuss please open issues in our Jira not github